packages/tabs/src/LionTabs.js

```diff
@@ -1,4 +1,4 @@
-import { LitElement, css, html } from '@lion/core';
+import { LitElement, css, html } from 'lit-element';
```

Comment:
Can you elaborate on this choice? 
Should we maybe do it in a seperate merge request for all packages?
Since we want to have all our packages work nicely together (reducing the risk of dependency conflicts) I like a centrally orchestrated version. (not saying @lion/core is the only way to achieve this...)

Reply:
this is to make it work for now...
as soon as you import from `@lion/core` all the typings fail as apparently the typings for lit-element are not "forwarded"

I do hope this get's fixed once `@lion/core` becomes typed as well

Reply:
Ah :)